src/components/table-static/TableStatic.vue

```diff
@@ -249,15 +277,89 @@ const indeterminate = computed(() => {
     && model.value.length < selectableRows.value.length
 })
 
+const defaultItems = ref<T[]>()
+const tableSorts   = ref<TableSort<T>[]>([])
+
+const itemsWatcher = watchPausable(() => props.items, (value) => {
+  if (props.sortable && !props.sortableAsyncronous) {
+    defaultItems.value = value
+    tableSorts.value   = []
+  }
+})
+
+async function sortField (field: TableField<T>) {
+  if (!props.sortable || !field.sortable || field.sortable === undefined)
```

Comment:
@adenvt 
I also added a check here, when the field is not sortable it will not be sorted